scripts/custom_scripts/090_update_basic_solr_config.sh

```diff
@@ -8,12 +8,17 @@ fi
 # Clone utk-fedora-fedoragsearch-solr-config
 echo "Cloning utk-fedora-fedoragsearch-solr-config"
 if [ ! -d "$HOME_DIR"/utk-fedora-fedoragsearch-solr-config ]; then
-  git clone https://github.com/utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config
-  chown -hR $HOMEUSER:$HOMEUSER utk-fedora-fedoragsearch-solr-config
+  git -C "$HOME_DIR" clone https://github.com/utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config
+  chown -hR $HOMEUSER:$HOMEUSER "$HOME_DIR"/utk-fedora-fedoragsearch-solr-config
+  git -C "$HOME_DIR"/utk-fedora-fedoragsearch-solr-config checkout -b institutional_repository origin/institutional_repository
```

Comment:
Looks good.  Verified that this is indeed what's coming in and loading in TRACE.